corehq/apps/export/tasks.py

```diff
@@ -26,16 +28,32 @@ def populate_export_download_task(export_instances, filters, download_id, filena
 
     file_format = Format.from_format(export_file.format)
     filename = filename or export_instances[0].name
-    payload = export_file.file.payload
-    expose_cached_download(
-        payload,
-        expiry,
-        ".{}".format(file_format.extension),
-        mimetype=file_format.mimetype,
-        content_disposition=safe_filename_header(filename, file_format.extension),
-        download_id=download_id,
-    )
-    export_file.file.delete()
+    domain = export_instances[0].domain
+
+    try:
+        if BLOBDB_EXPORTS.enabled(domain):
+            db = get_blob_db()
```

Comment:
@millerdev is this something that is better down at the module level rather than inside of a function?